Formula/ghc.rb

```diff
@@ -112,14 +87,13 @@ def install
       ENV.prepend_path "PATH", binary/"bin"
     end
 
-    args << "--with-intree-gmp" if OS.linux?
-
-    system "./configure", "--prefix=#{prefix}", *args
+    system "./configure", "--prefix=#{prefix}", "--disable-numa", "--with-intree-gmp", "--with-system-libffi"
```

Comment:
From local builds, GHC built `--with-system-libffi` may be propagating `/usr/lib/libffi.dylib` linkage. Would be more of an issue on Linux, but may just keep building the bundled libffi to avoid anything unexpected.